pkg/capacity/printer.go

```diff
@@ -63,10 +63,20 @@ func printList(cm *clusterMetric, showContainers, showPods, showUtil, showPodCou
 			showPodCount:    showPodCount,
 			showContainers:  showContainers,
 			showNamespace:   showNamespace,
+			hideRequests:    hideRequests,
+			hideLimits:     hideLimits,
 			sortBy:          sortBy,
 			w:               new(tabwriter.Writer),
 			availableFormat: availableFormat,
 		}
+		if !tp.hasVisibleColumns() {
+			fmt.Println("Error: No data columns selected for display. At least one of the following must be enabled:")
+			fmt.Println("- Resource requests (enabled by default, disabled with --no-requests)")
+			fmt.Println("- Resource limits (enabled by default, disabled with --no-limits)") 
+			fmt.Println("- Resource utilization (enabled with --util)")
+			fmt.Println("- Pod count (enabled with --pod-count)")
+			os.Exit(1)
+		}
 		tp.Print()
 	} else if output == CSVOutput || output == TSVOutput {
```

Comment:
It looks like this is intentionally omitted from CSV/TSV mode, mind documenting that in the readme? Ideally we'd also surface a warning if these flags were used with an unsupported output type.

Reply:
fixed https://github.com/robscott/kube-capacity/pull/172/commits/444667a951f04b8bcc195eaf8438e26c390f82ca